src/volatile_memory.rs

```diff
@@ -1574,6 +1576,7 @@ mod tests {
         // somehow and this helps to ensure the sleep actually happens before the store rather then
```

Comment:
I would update the comments, so they do not mention about `sleep` anymore.

Reply:
fixed